packages/builders/scripts/fast-usdc/add-operators.build.js

```diff
@@ -0,0 +1,83 @@
+// @ts-check
+import { makeHelpers } from '@agoric/deploy-script-support';
+import { getManifestForAddOperators } from '@agoric/fast-usdc/src/add-operators.core.js';
+import { toExternalConfig } from '@agoric/fast-usdc/src/utils/config-marshal.js';
+import { configurations } from '@agoric/fast-usdc/src/utils/deploy-config.js';
+import { Far } from '@endo/far';
+import { parseArgs } from 'node:util';
+
+/**
+ * @import {CoreEvalBuilder, DeployScriptFunction} from '@agoric/deploy-script-support/src/externalTypes.js';
+ * @import {ParseArgsConfig} from 'node:util';
+ * @import {FastUSDCConfig, FeedPolicy} from '@agoric/fast-usdc/src/types.js';
+ * @import {FastUSDCOpts} from './init-fast-usdc.js';
+ */
+
+const { keys } = Object;
+
+/** @type {ParseArgsConfig['options']} */
+const options = {
+  net: { type: 'string' },
+  oracle: { type: 'string', multiple: true },
+};
+const oraclesUsage = 'use --oracle name:address ...';
+
+const crossVatContext = /** @type {const} */ ({
+  /** @type {Brand<'nat'>} */
+  USDC: Far('USDC Brand'),
+});
+
+/** @type {CoreEvalBuilder} */
+export const defaultProposalBuilder = async (
+  powers,
+  /** @type {FastUSDCConfig} */ config,
+) => {
+  return harden({
+    sourceSpec: '@agoric/fast-usdc/src/add-operators.core.js',
+    /** @type {[string, Parameters<typeof getManifestForAddOperators>[1]]} */
+    getManifestCall: [
+      getManifestForAddOperators.name,
+      {
+        options: toExternalConfig(config, crossVatContext),
```

Comment:
You could skip `toExternalConfig` since this is JSON-happy data.

Reply:
understood. I try to keep the differences to line deletion as much as possible, for easier reconciliation